### PR TITLE
Numishaul balance changes (February 9th, 2024) and Lapidary Calibration overhaul

### PR DESCRIPTION
**Pull request by TrashboxBobylev**
This pull request does following things:

- rebalances RF fuel values on coins in accordance to updated Numishaul spreadsheet data, which has been updated to include automatic formulas for most of its calculations
- changes Mirion's amount of coin produced from one ingot from 3 to 1, to counteract it being obtainable from IC2 agriculture
- overhauls Lapidary Calibration augment's fuel list by making it include the majority of the game's gems and crystals and rebalancing existing gems:
javaw_1Oxd9683WZ
- adds a "mechanic" for allowing to use gem blocks in Numismatic Dynamo (with Lapidary Calibration augment) and making gem blocks give slightly more power than all gems used to make them, like how coal blocks burn for longer than 9 coal used to make them
javaw_n9MqKFG50T
javaw_v5VGhiVg80
- removed Emerald from unaugmented Numismatic Dynamo's fuel list, allowing it to be only used as gem fuel
- adds new villager type, called Stock Trader, that allows to buy and exchange coins for better coins
javaw_R53dXgtXOp

### CODE REVIEW

File: resources/vtt/lang/en_US.lang
Comment (Krutoy242) [inline resources/vtt/lang/en_US.lang:1]: Please add ru_ru and zh_cn files at least placeholders, but better translated. "Stock Trader" is common term so should be no problem.

File: config/vtt/trade_tables/stock_trader.json
Comment (Krutoy242) [inline config/vtt/trade_tables/stock_trader.json:1]: Please format JSON file. VSCode have many handy formatters. Right now there is `\t` and spaces together.

File: scripts/cot/cot_coins.zs
Comment (Krutoy242) [inline scripts/cot/cot_coins.zs:1]: Please unify tabs to be 2 space symbols. Right now part of the file content tabs instead of 2 spaces.

**Review by Krutoy242 — CHANGES_REQUESTED** [review]

**Review by Krutoy242 — APPROVED** [review]

File: resources/vtt/lang/ru_RU.lang
Comment (Krutoy242) [inline resources/vtt/lang/ru_RU.lang:1]: Броке? Я бы принял если бы ты русского не знал 😂

**Review by Krutoy242 — APPROVED** [review]

File: resources/vtt/lang/ru_RU.lang
Comment (TrashboxBobylev) [reply resources/vtt/lang/ru_RU.lang:1]: https://ru.wikipedia.org/wiki/%D0%91%D1%80%D0%BE%D0%BA%D0%B5%D1%80

Comment (TrashboxBobylev) [reply resources/vtt/lang/ru_RU.lang:1]: увидел...

**Pull request merged**